src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int RATE_INDEX = 2;
+    private static final int TIME_INDEX = 3;
+    private static final String DASH = " - ";
+    private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
For the future, this variable can be a constant since we do not change it
```suggestion
    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```